src/main/java/org/generationcp/middleware/service/DataImportServiceImpl.java

```diff
@@ -1164,24 +1164,7 @@ private void checkForExistingTrialInstance(final Workbook workbook, final Map<St
 				}
 			}
 		}
-		boolean hasDuplicateTrialInstances = false;
 		if (!duplicateTrialInstances.isEmpty() && workbook.getStudyDetails().getId() != null) {
-			// check import type first
-			final List<Integer> variateIds = new ArrayList<>();
-			// check all variates
-			for (final MeasurementVariable mvar : workbook.getVariates()) {
-				variateIds.add(mvar.getTermId());
-			}
-
-			final int numberOfVariatesData = this.getPhenotypeDao()
-				.countVariatesDataOfStudy(
-					isMeansDataImport ? workbook.getMeansDatasetId() : workbook.getMeasurementDatesetId(),
-					variateIds);
-			if (numberOfVariatesData > 0) {
-				hasDuplicateTrialInstances = true;
-			}
-		}
-		if (hasDuplicateTrialInstances) {
```

Comment:
As discussed, we need to handle still being able to import means dataset too for the same instance number.